src/web/app/pages-admin/admin-search-page/admin-search-page.component.ts

```diff
@@ -59,11 +60,16 @@ export class AdminSearchPageComponent {
         this.statusMessageService.showWarningToast('No results found.');
         this.instructors = [];
         this.students = [];
-      } else {
-        this.instructors = resp.instructors;
-        this.students = resp.students;
-        this.hideAllInstructorsLinks();
-        this.hideAllStudentsLinks();
+        return;
+      }
+
+      this.instructors = resp.instructors;
+      this.students = resp.students;
+      this.hideAllInstructorsLinks();
+      this.hideAllStudentsLinks();
+      if (this.instructors.length + this.students.length >= this.maxResultsToShow) {
+        this.statusMessageService.showWarningToast(`${this.maxResultsToShow} results have been shown on this page
+            but there may be more results not shown. Consider searching with more specific termss.`);
```

Comment:
There's a typo here: extra 's' in 'termss'

Reply:
@jianhandev Thank you for catching that!